src/Elmish.WPF/DynamicViewModel.fs

```diff
@@ -55,31 +56,32 @@ type [<AllowNullLiteral>] internal DynamicViewModel<'model, 'msg>
     log.LogTrace("[{BindingNameChain}] ErrorsChanged {BindingName}", nameChain, name)
     errorsChanged.Trigger([| box this; box <| DataErrorsChangedEventArgs name |])
 
-  let (bindings, validationErrors) =
+  let getFunctionsForSubModelSelectedItem (selectedItemTargetBindings: IReadOnlyDictionary<string, _>) name =
+    selectedItemTargetBindings
+    |> IReadOnlyDictionary.tryFind name
+    |> function
+      | Some b ->
+        match FuncsFromSubModelSeqKeyed().Recursive(b) with
+        | Some x -> Some x
+        | None -> log.LogError("SubModelSelectedItem binding referenced binding {SubModelSeqBindingName} but it is not a SubModelSeq binding", name)
+                  None
+      | None -> log.LogError("SubModelSelectedItem binding referenced binding {SubModelSeqBindingName} but no binding was found with that name", name)
+                None
+
+  let initializeBindingWithValidation selectedItemTargetBindings binding =
```

Comment:
This diff is bad.  It took me a long time to notice that you added `selectedItemTargetBindings` here.  When you move code, please do not otherwise chang it.  If it much change, please consider pointing out what changed.

Reply:
Ok I have it further separated and put a small note in the git commit comments. Do you want me to make separate PR's for the commits?